src/box/memtx_engine.cc

```diff
@@ -1265,6 +1265,84 @@ struct memtx_join_ctx {
 	struct xstream *stream;
 };
 
+static void
+send_join_meta(struct xstream *stream, const struct vclock *vclock,
+	       const bool send_meta, const struct raft_request *raft_req,
+	       const struct synchro_request *synchro_req)
```

Comment:
This function was introduced in a previous commit and now you move it. Please don't do that. Instead, add it to the right place in the commit that introduced it.

Reply:
My bad, fixed